Webapp/sources/services/email.py

```diff
@@ -4,11 +4,128 @@
 import jwt
 from flask import current_app, render_template
 from flask_login import current_user
-
+from qrcode import QRCode
 from sources import models, services
 from sources.extensions import mail
 
-from qrcode import QRCode
+
+def send_reaction_approval_request(
+    user: models.User,
+    reaction_approval_request: models.ReactionApprovalRequest,
+    workgroup: models.WorkGroup,
+    workbook: models.WorkBook,
+    reaction: models.Reaction,
+):
+    """
+    Send reaction approval request email to a given user. They have 70 days to respond
+    Args:
+        user (models.User): User to send to.
+        reaction_approval_request (models.ReactionApprovalRequest): Reaction approval request.
+        workgroup (models.WorkGroup): Workgroup the reaction belongs to
+        workbook (models.WorkBook): Workbook the reaction belongs to
+        reaction (models.Reaction): The reaction for approval
+    """
+    token = get_encoded_token(
+        time_limit=6048000,  # 70 days?
```

Comment:
If the time_limit is in seconds you could to `60 * 60 * 24 * 70`. Might make it a little more obvious it's meant to be 70 days